.github/workflows/pr_eaa.yml

```diff
@@ -0,0 +1,144 @@
+
+    - name: install rats-tls & verdictd
+      run: docker exec $inclavare_test bash -c 'mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /usr/local/lib/;
+        mv /root/inclavare-containers/${{ matrix.tag }}/libopa.so /usr/local/lib/;
+        mv /root/inclavare-containers/${{ matrix.tag }}/verdictd /usr/local/bin/'
+
+    - name: install sample image
+      run: docker exec $inclavare_test bash -c 'mkdir /root/images && cd /root/images;
+        skopeo --insecure-policy copy docker://docker.io/library/alpine:latest oci:alpine'
+
+    - name: install configuration files
+      run: |
+        docker exec $inclavare_test bash -c 'mkdir /opt/verdictd/ && mkdir /opt/verdictd/keys/'
+        docker exec $inclavare_test bash -c "cat <<- EOF >/opt/verdictd/keys/${{ env.UUID }}
+        11111111111111111111111111111111
+        EOF"
+        
+        docker exec $inclavare_test bash -c "cat <<- EOF >/root/images/pod.yaml
+        metadata:
+          attempt: 1
+          name: my-podsandbox
+          namespace: default
+          uid: hdishd83djaidwnduwk28bcsb
+        log_directory: /tmp/eaa_test
+        linux:
+          namespaces:
+            options: {}
+        EOF"
+        
+        docker exec $inclavare_test bash -c "cat <<- EOF >/root/images/container.yaml
+        metadata:
+          name: alpine.enc
+        image:
+          image: ${{ secrets.DOCKERHUB_NAME }}/${{ env.ENCRYPTED_IMAGE }}-${{ matrix.tag }}:latest
+        command:
+        - top
+        log_path: busybox.0.log
+        EOF"
+        
+        docker exec $inclavare_test bash -c 'rm -fr /etc/containerd/ocicrypt/ocicrypt_keyprovider.conf'
+        docker exec $inclavare_test bash -c "cat <<- EOF >/etc/containerd/ocicrypt/ocicrypt_keyprovider.conf
+        {
+            \"key-providers\": {
+                \"eaa\": {
+                    \"grpc\": \"127.0.0.1:${{ steps.random-port-generator1.outputs.random-port }}\"
+                }
+            }
+        }
+        EOF"
+
+    - name: Launch all Services
+      run: |
+        docker exec $inclavare_test bash -c 'verdictd --listen 127.0.0.1:${{ steps.random-port-generator2.outputs.random-port }} --gRPC 127.0.0.1:${{ steps.random-port-generator1.outputs.random-port }} --config 127.0.0.1:${{ steps.random-port-generator3.outputs.random-port }}' &
+        docker exec $inclavare_test bash -c 'containerd' &
+
+    - name: Generate encrypted container image
+      run: |
+        docker exec $inclavare_test bash -c 'cd /root/images/ && OCICRYPT_KEYPROVIDER_CONFIG=/etc/containerd/ocicrypt/ocicrypt_keyprovider.conf skopeo copy --insecure-policy --encryption-key provider:eaa:${{ env.UUID }} oci:alpine oci:${{ env.ENCRYPTED_IMAGE }}-${{ matrix.tag }}' 
+
+    - name: Push encrypted container image to Docker Hub
+      run: |
+        docker exec $inclavare_test bash -c 'skopeo login docker.io --username ${{ secrets.DOCKERHUB_NAME }} --password ${{ secrets.DOCKERHUB_PASSWORD }};
```

Comment:
Please refer to [link](https://github.com/alibaba/inclavare-containers/blob/master/.github/workflows/manually_compilation_testing_image.yml#L29) to push test images to runetest dockerhub.